dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
will we settle on one or the other over time, or will we always have two possibilities here?